### ISSUE TITLE
Documenter Docs

### ISSUE THREAD

**Issue by joshday** [JuliaComputing/PlotlyLight.jl#46]
README is getting unruly and we could use some proper documentation.

**Comment by abcdvvvv** [comment]
Yes, we need a document for Julia users. As a noob, I don't know how this Julia interface to JavaScript is done, so I can't read the JavaScript documentation.
Maybe there should be a documentation for the package itself.
